common/crypto/dkg/src/bte/encryption.rs

```diff
@@ -24,7 +24,8 @@ pub struct Ciphertexts {
 }
 
 impl Ciphertexts {
-    pub fn verify_integrity(&self, params: &Params, epoch: Epoch) -> bool {
+    pub fn verify_integrity(&self, params: &Params) -> bool {
+        let epoch = Epoch::default();
```

Comment:
I thought we wanted to get rid of `Epoch` completely? 
I might be misremembering (you'd have to try and run unit tests to see if it still passes), but you might be able to get rid of /simplify`Tau` conversion since you're just going to end up with 0 bits